redash/query_runner/mixpanel.py

```diff
@@ -0,0 +1,198 @@
+            except urllib2.URLError as e:
+                Mixpanel.LOGGER.warning('We failed to reach a server.')
+                Mixpanel.LOGGER.warning('Reason: ' + str(e.reason))
+                if hasattr(e, 'read'):
+                    Mixpanel.LOGGER.warning('Response: ' + e.read())
+                Mixpanel.LOGGER.warning("Attempting retry #" + str(retries + 1))
+                self.request(base_url, path_components, params, method=method, headers=headers, raw_stream=raw_stream,
+                             retries=retries + 1)
+            except SSLError as e:
+                if e.message == 'The read operation timed out':
+                    Mixpanel.LOGGER.warning('The read operation timed out.')
+                    self.timeout = self.timeout + 30
+                    Mixpanel.LOGGER.warning(
+                        'Increasing timeout to ' + str(self.timeout) + ' and attempting retry #' + str(retries + 1))
+                    self.request(base_url, path_components, params, method=method, headers=headers,
+                                 raw_stream=raw_stream, retries=retries + 1)
+                else:
+                    raise
+            else:
+                try:
+                    # If the response is gzipped we go ahead and decompress
+                    if response.info().get('Content-Encoding') == 'gzip':
+                        buf = cStringIO.StringIO(response.read())
+                        f = gzip.GzipFile(fileobj=buf)
+                        response_data = f.read()
+                    else:
+                        response_data = response.read()
+                    return response_data
+                except IncompleteRead as e:
+                    Mixpanel.LOGGER.warning("Response data is incomplete. Attempting retry #" + str(retries + 1))
+                    self.request(base_url, path_components, params, method=method, headers=headers,
+                                 raw_stream=raw_stream, retries=retries + 1)
+        else:
+            Mixpanel.LOGGER.warning("Maximum retries reached. Request failed. Try again later.")
+            raise BaseException
+
+
+def flatten_row(row):
+    result = {}
+    for k in row:
+        if type(row[k]) is dict:
+            for j in row[k]:
+                result[k + "_" + j] = row[k][j]
+        elif type(row[k]) is list:
+            for index, j in enumerate(row[k]):
+                result[k + "_" + str(index)] = row[k][index]
+        else:
+            result[k] = row[k]
+    return result
+
+
+class MixpanelJql(BaseQueryRunner):
+    noop_query = '''
+        main = () => People().reduce(mixpanel.reducer.null())
+    '''
+
+    @classmethod
+    def annotate_query(cls):
+        return False
+
+    @classmethod
+    def type(cls):
+        return "mixpanel_jql"
+
+    @classmethod
+    def name(cls):
+        return "Mixpanel JQL"
+
+    def __init__(self, configuration):
+        super(MixpanelJql, self).__init__(configuration)
```

Comment:
This is the default behavior, so you can just remove the `__init__` method here.